docs/conf.py

```diff
@@ -196,7 +196,11 @@
 .. raw:: html
 
     <div>
-      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a></p>
+      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a>
```

Comment:
An alternative to the other two minor tweaks, would be to rephrase this to be one longer line, something like:

```html
<p>
  Execute this notebook: 
  <a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download locally</a>
  <a href="https://mybinder.org/v2/gh/stellargraph/stellargraph/master?urlpath=lab/tree/{{ env.docname }}.ipynb" alt="Open In Binder"><img src="https://mybinder.org/badge_logo.svg"/></a>
  <a href="https://colab.research.google.com/github/stellargraph/stellargraph/blob/master/{{ env.docname }}.ipynb" alt="Open In Colab"><img src="https://colab.research.google.com/assets/colab-badge.svg"/></a>
</p>
```

That looks something like:

![image](https://user-images.githubusercontent.com/1203825/80945098-63123980-8e2e-11ea-8cb6-de2bbec3361d.png)

Reply:
I like this version grouping them